File: VB/MyLayoutView/MyLayoutViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid.Views.Layout.ViewInfo
Imports DevExpress.XtraGrid.Views.Layout
Imports System.Drawing
Imports DevExpress.XtraGrid.Views.Grid.ViewInfo

Namespace MyXtraGrid
	Public Class MyLayoutViewInfo
		Inherits LayoutViewInfo
		Public Sub New(ByVal view As DevExpress.XtraGrid.Views.Layout.LayoutView)
			MyBase.New(view)

		End Sub
	End Class
End Namespace

File: CS/MyLayoutView/MyLayoutViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid.Views.Layout.ViewInfo;
using DevExpress.XtraGrid.Views.Layout;
using System.Drawing;
using DevExpress.XtraGrid.Views.Grid.ViewInfo;

namespace MyXtraGrid
{
    public class MyLayoutViewInfo : LayoutViewInfo
    {
        public MyLayoutViewInfo(DevExpress.XtraGrid.Views.Layout.LayoutView view)
            : base(view)
        {
            
        }
    }
}
